mlflow/environment_variables.py

```diff
@@ -536,3 +536,8 @@ def get(self):
 #: The OpenJDK version to install in the Docker image used for MLflow models.
 #: (default: ``11``)
 MLFLOW_DOCKER_OPENJDK_VERSION = _EnvironmentVariable("MLFLOW_DOCKER_OPENJDK_VERSION", str, "11")
+
+MLFLOW_UNITY_CATALOG_PRESIGNED_URLS_ENABLED = _BooleanEnvironmentVariable(
```

Comment:
How will this be changed?  Do we have an existing pattern of doing this in mlflow?

Reply:
When the backend changes are implemented, this will be defaulted to `True`.  The environment variable will remain as a fallback in case problems are encountered with the `PresignedUrlArtifactRepository`. There is a history of doing this in mlflow, per offline discussion with @smurching.